src/core/ext/xds/xds_client_stats.h

```diff
@@ -63,7 +63,11 @@ class XdsLocalityName : public RefCounted<XdsLocalityName> {
   XdsLocalityName(std::string region, std::string zone, std::string sub_zone)
       : region_(std::move(region)),
         zone_(std::move(zone)),
-        sub_zone_(std::move(sub_zone)) {}
+        sub_zone_(std::move(sub_zone)),
+        locality_labels_(
+            std::make_shared<std::map<std::string, std::string>>()) {
+    locality_labels_->emplace("grpc.lb.locality", AsHumanReadableString());
```

Comment:
is this the format that we want the labels to be in?

Reply:
In general, I think the optional label keys should be the exact name of the label that we want to appear on the metrics, so that stats plugins (a) don't have to any transformations and (b) can more easily enable the same optional label for both per-call and non-per-call metrics.

We didn't get this quite right for the CSM labels, unfortunately, so we're stuck with a weird edge case there, but we should do the right thing for all subsequent labels.

Reply:
I forgot about this comment. I meant whether the value should be the thing provided by `AsHumanReadableString`.

Reply:
Is there some reason to use a different value?

Reply:
That has a format of `"{region=\"%s\", zone=\"%s\", sub_zone=\"%s\"}"`

If that's the format we want, shouldn't we add it in https://github.com/grpc/proposal/pull/419/files?